src/main/java/frc/utils/applicationsutils/CMDHandler.java

```diff
@@ -23,15 +25,25 @@ public static void runCMDCommand(Path directory, String command) {
     public static void runCMDCommand(String command) {
         String cmdSpecification = isWindows() ? WINDOWS_CMD_SPECIFICATION : NON_WINDOWS_CMD_SPECIFICATION;
         Runtime runtime = Runtime.getRuntime();
-        System.out.println("Running: " + cmdSpecification + command);
+        command = cmdSpecification + command;
```

Comment:
i prefer it separate

Reply:
its a lot of repeating that though. It is better to just save that variable then add them all of the time.

Reply:
but its like you tell the shell with /c what to do with the string after it which is the command, the shell is not the command itself

Reply:
oh ok